pkg/operators/wasm/maps.go

```diff
@@ -76,6 +78,10 @@ func (i *wasmOperatorInstance) addMapFuncs(env wazero.HostModuleBuilder) {
 	)
 }
 
+// Keep in sync with wasmapi/go/map.go.
+// Get the next free flag after UpdateLock
+var getFdFlag = uint64(1 << (bits.TrailingZeros64(uint64(ebpf.UpdateLock)) + 1))
```

Comment:
What about keeping space for potential new upstream flags after `BPF_F_LOCK`?
```suggestion
var getFdFlag = uint64(1 << 32)
```